app/scripts/modules/core/src/cluster/ClusterPostSearchResultSearcher.ts

```diff
@@ -25,13 +21,15 @@ export class ClusterPostSearchResultSearcher implements IPostSearchResultSearche
     return { account, application, cluster, provider, stack, displayName: cluster, href, type };
   }
 
-  public getPostSearchResults(inputs: IServerGroupSearchResult[] = []): IPromise<ISearchResultSet[]> {
+  public getPostSearchResults(resultSet: ISearchResultSet<IServerGroupSearchResult>): IPromise<ISearchResultSet> {
+    const inputs = resultSet.results;
     const type: ISearchResultType = searchResultTypeRegistry.get(this.TYPE_ID);
 
     // create cluster search results based on the server group search results
     const clusters = inputs.map(input => this.makeSearchResult(input));
     const results: IClusterSearchResult[] = uniqBy(clusters, sg => `${sg.account}-${sg.cluster}`);
```

Comment:
we can do something like this: ```Array.from(new Set(clusters.map(sg => `${sg.account}-${sg.cluster}`)))``` to avoid using uniqBy from lodash.

Reply:
I don't think that would work here, as you'd end up with an array of strings. 

`uniqBy` lets you drop elements from an array that are duplicates based on a property or computation.

Reply:
Oh ok. A .filter method can help avoid duplicates, but it will have to be its own method. I was hoping to avoid loading lodash. But looks like we use it in a few other places too.

Reply:
we use lodash quite a lot 😐

Reply:
We used to use lodash a lot more than we do now, but it's still very nice for things like debouncing, getting default values, deep property inspection via `get` and `has`, among others.